native/SampleApps/MobileSyncExplorer/MobileSyncExplorer.xcodeproj/project.pbxproj

```diff
@@ -1157,8 +1158,9 @@
 				OTHER_LDFLAGS = "-ObjC";
 				PRODUCT_BUNDLE_IDENTIFIER = com.salesforce.mobilesdk.MobileSyncExplorer;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				SUPPORTS_MACCATALYST = YES;
+				SUPPORTS_MACCATALYST = NO;
```

Comment:
Why not?

Reply:
I enabled it in my last PR but I didn't get a chance to update the UI, so I turned it off again. It'll still be enabled in RestAPIExplorer and MobileSyncExplorerSwift template